Panels/Panel.js

```diff
@@ -272,6 +283,17 @@ const PanelDecorator = hoc({defaultElement: `.${componentCss.body} *`}, (config,
  * @memberof sandstone/Panels.Panel.prototype
  */
 
+/**
+ * The container id for {@link spotlight/Spotlight}.
+ *
+ * When used within [Panels]{@link sandstone/Panels.Panels}, this prop may be set by `Panels`.
```

Comment:
When the `Panel` is used within [Panels]{@link sandstone/Panels.Panels}, this prop will be generated by `Panels`.
When using the panel alone, you need to specify this prop for restoring focus.

Reply:
Thanks I fixed